iptables/_rule.sls

```diff
@@ -11,6 +11,9 @@ iptables_{{ chain_name }}_{{ rule_name }}:
   {%- endif %}
   - table: {{ rule.get('table', 'filter') }}
   - chain: {{ chain_name }}
+  {%- if rule.family is defined %}
+  - family: {{ rule.family }}
```

Comment:
It would be good to support list of families or `family: all` to avoid defining same rules for both protocols.